lib/Kohana.js

```diff
@@ -33,13 +32,20 @@ export default class Kohana extends BaseInput {
     /*
      * Passed to react-native-vector-icons library as size prop.
      */
-    iconSize: PropTypes.number,
+	iconSize: PropTypes.number,
```

Comment:
Please fix indentation problems. The library uses 2 whitespaces for indentation.